.prettierignore

```diff
@@ -0,0 +1,5 @@
+tests/graphql/inline-many.graphql
+tests/graphql/inline-many-list.graphql
+tests/graphql/test-inline.graphql
+tests/graphql/test-inline-list.graphql
+tests/graphql/test-server-vars.graphql
```

Comment:
Why are we ignoring these files?

Reply:
Prettier puts space between array elements in these files and I couldn't find any way to change this action other than ignoring it. I tried changing the actual and expected logic for the test but I couldn't figure it out either :(